app/components/modal/food/food.html

```diff
@@ -0,0 +1,40 @@
+<div class="modal-header">
+    <button type="button" class="close" ng-click="foodCtrl.modalCancel()">&times;</button>
+    <h3 ng-hide="foodCtrl.copyFood" class="modal-title">Add Food</h3>
+    <h3 ng-show="foodCtrl.copyFood" class="modal-title">Edit Food</h3>
```

Comment:
this can be done in single line.
<h3 class="modal-title">{{foodCtrl.copyFood?'Edit':'Add'}} Food</h3>